### PR TITLE
Feature/admin resources

### PR DESCRIPTION
**Pull request by minahil3955**
Admin Resources
1. Add Project-Leads
2. Add Projects 
3. Add Client
4. Add Phases

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -35,6 +35,8 @@ gem 'devise'
 
 gem 'pundit'
 
+gem 'activeadmin'
```
Comment (muhammad-ahmad-shafiq) [inline Gemfile]: Why did we add it 2 times?

File: app/admin/clients.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+ActiveAdmin.register Client do
+  permit_params :name, :address, :email, :contact, project_leads_attributes: [:name]
+  index do
```
Comment (muhammad-ahmad-shafiq) [inline app/admin/clients.rb:4]: Leave a space before `index do`

File: app/admin/phases.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+ActiveAdmin.register Phase do
+  permit_params :due_date, :status, :name, :project_lead_id
+  form do |f|
```
Comment (muhammad-ahmad-shafiq) [inline app/admin/phases.rb:3]: leave a space after `permit_params` line

File: app/admin/project_leads.rb [hunk 4]
```diff
@@ -0,0 +1,12 @@
+ActiveAdmin.register ProjectLead do
+  permit_params :user_id, :client_id, :name, :platform, :is_sale
+  form do |f|
```
Comment (muhammad-ahmad-shafiq) [inline app/admin/project_leads.rb:4]: same

File: app/admin/projects.rb [hunk 5]
```diff
@@ -0,0 +1,3 @@
+ActiveAdmin.register Project do
+  permit_params :name
```
Comment (muhammad-ahmad-shafiq) [inline app/admin/projects.rb:2]: If there is no form, then why are we permitting a param? If this is not finalized or finished, then delete this file for now, you can add this later

File: config/initializers/active_admin.rb [hunk 6]
```diff
@@ -4,6 +4,7 @@
   # Set the title that is displayed on the main layout
   # for each of the active admin pages.
   #
+
```
Comment (muhammad-ahmad-shafiq) [inline config/initializers/active_admin.rb]: Why 👁️

File: config/routes.rb [hunk 7]
```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
   devise_for :admin_users, ActiveAdmin::Devise.config
   ActiveAdmin.routes(self)
+
```
Comment (muhammad-ahmad-shafiq) [inline config/routes.rb]: again, why 👀

**Review by muhammad-ahmad-shafiq — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 1 above)
Comment (minahil3955) [reply Gemfile]: It was left while resolving conflicts, will remove it.

File: app/admin/projects.rb (hunk 5 above)
Comment (minahil3955) [reply app/admin/projects.rb:2]: This is built in, I have a page of Projects at admin side of this.

File: config/initializers/active_admin.rb (hunk 6 above)
Comment (minahil3955) [reply config/initializers/active_admin.rb]: Removed

File: config/routes.rb (hunk 7 above)
Comment (minahil3955) [reply config/routes.rb]: done xD

File: app/admin/phases.rb (hunk 3 above)
Comment (minahil3955) [reply app/admin/phases.rb:3]: done

**Review by muhammad-ahmad-shafiq — APPROVED** [review]

**Pull request merged**